test/modules/standard/List/memory.chpl

```diff
@@ -0,0 +1,19 @@
+use Memory;
+use List;
+
+var mem : uint;
+proc output()
+{
+	mem=memoryUsed();
+	var ls : list(int);
+	ls.push_back(5);
+	ls.push_back(8);
+	ls.push_back(10);
+	ls.push_back(12);
+	ls.push_back(15);
+}
+
+memTrack;
+writeln(memoryUsed());
+output();
+writeln(mem-memoryUsed());
```

Comment:
This subtraction is backwards.  If memory were leaked (as it is on master), memoryUsed() would be > mem.